test/testingUtils.py

```diff
@@ -0,0 +1,59 @@
+import json
+from algosdk import mnemonic
+from algosdk.error import IndexerHTTPError
+from algosdk.v2client import indexer
+from akita_inu_asa_utils import wait_for_txn_confirmation, \
+    payment_signed_txn, \
+    get_algod_client
+import time
+
+
+def load_test_config(file_path='./test/testConfig.json'):
+    fp = open(file_path)
+    return json.load(fp)
+
+
+def fund_account(address, initial_funds=2000000):
+    test_config = load_test_config('./test/testConfig.json')
+    client = get_algod_client(test_config['algodToken'], test_config['algodAddress'])
+    private_key = mnemonic.to_private_key(test_config['fund_account_Mnemonic'])

```

Comment:
This config key isn't in the current test config template - however I think it would make more sense to pass in the mnemonic in the args